lighty-modules/lighty-netconf-sb/src/main/java/io/lighty/modules/southbound/netconf/impl/util/NetconfConfigUtils.java

```diff
@@ -118,21 +174,51 @@ public static NetconfConfiguration createDefaultNetconfConfiguration() {
         return new NetconfConfiguration();
     }
 
+    /**
+     * Create default Netconf Southbound configuration, with custom dispatcher and encryptionService.
+     *
+     * @param encryptionService custom AAAEncryptionService provided by user to be injected.
+     * @param clientDispatcher dispatcher from running lighty services.
+     * @return Object representation of configuration data.
+     */
+    public static NetconfConfiguration getDefaultNetconfConfiguration(final NetconfClientDispatcher clientDispatcher,
+                                                                      final AAAEncryptionService encryptionService) {
+        final NetconfConfiguration configuration = new NetconfConfiguration();
+        configuration.setAaaService(encryptionService);
+        configuration.setClientDispatcher(clientDispatcher);
+        return configuration;
+    }
+
     /**
      * Inject services from LightyServices to Netconf southbound configuration.
      *
      * @param configuration Netconf southbound configuration where should be services injected.
      * @return Netconf southbound configuration with injected services from Lighty core.
      * @throws ConfigurationException in case provided configuration is not valid.
      */
-    public static NetconfConfiguration injectServicesToConfig(
-            final NetconfConfiguration configuration) throws ConfigurationException {
+    @Deprecated
+    public static NetconfConfiguration injectServicesToConfig(final NetconfConfiguration configuration)
+            throws ConfigurationException {
         final AAAEncryptionService aaa = NetconfConfigUtils.createAAAEncryptionService(
                 getDefaultAaaEncryptServiceConfig());
         configuration.setAaaService(aaa);
         return configuration;
     }
 
+    /**
+     * Inject services from LightyServices to Netconf southbound configuration.
```

Comment:
This does not inject them from LightyServices.. there is no LightyServices reference coming in. Also I think this method is pretty useless, user can do the same thing with one line.